docs/paas/deployment_tools.md

```diff
@@ -0,0 +1,280 @@
+---
+id: deployment-tools
+title:  Deployment tools
+sidebar_label: Deployment tools
+---
+
+# Deployment tools
+
+Kubernetes offers, by design, two deployment strategies:
+
+* Recreate &rarr; terminates the pod and, once terminated, scales up the new one;
+* Rolling Update (the default one) &rarr; the new pod is scaled up and, when ready and healthy, the old one is terminated;
+
+look at the [official documentation](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) for more information.
+
+Sometimes the user needs other deployment strategies, not supported by design using Kubernetes.
+For this reason we analyzed some tool to fulfil this need:
+
+* [Flux - Flagger](https://flagger.app/);
+* [Argo - Argo-rollouts](https://argo-rollouts.readthedocs.io/).
+
+:::warning
+The tools above are not fully integrated into the Mia-Platform Console. The user can handle resources through the console and the projects configurations, but not through the Mia-Platform Console UI with a no-code approach.
```

Comment:
What is the purpose of this warning?

Reply:
The purpose is to warn the user that, at the moment, the Console UI doesn't have a dedicated page to graphically handle these tools configurations